charts/sentry/templates/pvc-geoip.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.geodata.accountID }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: data-sentry-geoip
+  labels:
+    app: sentry
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  accessModes:
+    - ReadWriteOnce
```

Comment:
In this volume access mode, all pods must be on the same node. You can change it to RWM, but then you need to specify in the readme that the storage provider must support this mode

Or download the database every time and save it to an ephemeral volume via the init container

Reply:
@adonskoy  I made rwx with a note in the readme.  The EmptyDir storage will need to be initialized in each service of the container, and plus it still uses memory, which can, in theory, lead to an oom kill. Which is not very good.